config/default.yml

```diff
@@ -1643,13 +1643,15 @@ Metrics/BlockLength:
   Description: 'Avoid long blocks with many lines.'
   Enabled: true
   VersionAdded: '0.44'
-  VersionChanged: '0.58'
+  VersionChanged: '0.66'
```

Comment:
📝 I change `VersionChanged` to change the default setting. I think that the next release when merging this will be 0.66.0.